### ISSUE TITLE
dotify github does not pick up on removed dotfiles

### ISSUE THREAD

**Issue by JoelQ** [mdespuits/dotify#12]
The following sequence of events is giving me an issue:
- I removed a dotfile using `dotify unlink`. 
- I used `dotify save` 
- I got a message that said _No files have been changed in Dotify. Would you like to push these changed to Github? [Yn]_
- I said `Y`
- I get a message that said _Pushing up to Github... Successfully pushed!_

However the changes did not show up in my repo. I went to the `.dotify` directory and ran `git status`. Sure enough, the changes were there but hadn't been staged or commited. I had to do that manually.

In a nutshell:  `dotify save` does not catch dotfiles removed with `dotify unlink`

**Comment by mdespuits** [comment]
Yes, I am familiar with this issue as I have run into it a few times myself. I am currently in the process of re-writing a lot of the backend powering the CLI (see [master..cli-refactor](https://github.com/mattdbridges/dotify/compare/master...cli-refactor) diff). There is still quite a bit to be done, but this will (I hope) prove to be included in the next minor release.

So for now, you still have to do your git removals by hand...sorry 'bout that.

**Comment by mdespuits** [comment]
See the [CHANGELOG](https://github.com/mattdbridges/dotify/blob/master/CHANGELOG.md) for upcoming work on this issue.
